src/index.html

```diff
@@ -12,30 +12,51 @@
 </div>
 
 <main>
+	<!--
+	You have two variables at your disposal:
+
+	fontdata: this is all the data in fontdata.json
+	fonts: these are additional details about the font,
+	       like charset, variable axes and instances.
+	-->
+
+	<!--
+	See fontdata.json and change the order of the fonts to
+	match the family structure. E.g. light → regular → bold.
+
+	You can specify which font is the "main" font (usually
+	the "regular" style), by assigning its ID to the main_id
+	variable. Note that counting starts at 0!
+	-->
+	{% assign main_id = 2 %}
```

Comment:
Could this part of the data files? All other configuration happens via the data files.
Or if not, I think it should be added to the README:

```
* Put your WOFF2 fonts in src/fonts and run yarn fontdata.
* Configure the site in src/_data/site.js.
* In JavaScript, respond to fail/success of loading the font in the FontFaceObserver code.
* In HTML/CSS, use the .variable-support element to communicate when variable fonts aren't supported.
* Start/stop heavy animations by using the .am-i-in-view and .in-view classes.
```

Reply:
Good call, I meant to update the README.

Re: separate config file for the `main_id` stuff, that might be neater but I'd have to introduce a new file, or add it to the `site.js` config where it might be overlooked 🤔